src/plugins/mf_priority.cpp

```diff
@@ -109,6 +109,13 @@ int64_t priority_calculation (flux_plugin_t *p, int urgency)
     queue_factor = b->queue_factor;
     bank_factor = b->bank_factor;
 
+    // pack fairshare using memo event
+    flux_jobtap_event_post_pack (p,
+                                 FLUX_JOBTAP_CURRENT_JOB,
+                                 "memo",
+                                 "{s:f}",
+                                 "fairshare", b->fairshare);
```

Comment:
_Probably_ should check the return code here and raise an error if fails (or add a comment that the error is purposefully ignored)

Reply:
Yeah, that sounds fine to me. Put a note that the error shouldn't be fatal in a comment too.